internal/states/statemgr/filesystem.go

```diff
@@ -92,6 +92,16 @@ func NewFilesystemBetweenPaths(readPath, writePath string) *Filesystem {
 	}
 }
 
+// SaveErroredTestStateFile saves the state present in memory as a stateFile in disk
+// and is invoked when "tofu test" fails to destroy resources.
+func SaveErroredTestStateFile(stateFilePath string, state *states.State) error {
+	log.Printf("Writing state to file: errored_test.tfstate")
```

Comment:
I don't think this log message should be the responsibility of the filesystem logic. we only seem to log inside filesystem for trace logs.

I would recommend moving this log up a level, and using the stream to write direct to the view.

Reply:
done